src/libraries/System.IO.MemoryMappedFiles/tests/MemoryMappedViewStream.Tests.cs

```diff
@@ -56,24 +57,33 @@ public void InvalidArguments()
             }
         }
 
+        public static IEnumerable<object[]> AccessLevelCombinationsData()
+        {
+            yield return new object[] { MemoryMappedFileAccess.ReadWriteExecute, MemoryMappedFileAccess.Read };
+            yield return new object[] { MemoryMappedFileAccess.ReadWriteExecute, MemoryMappedFileAccess.Write };
+            yield return new object[] { MemoryMappedFileAccess.ReadWriteExecute, MemoryMappedFileAccess.ReadWrite };
+            yield return new object[] { MemoryMappedFileAccess.ReadWriteExecute, MemoryMappedFileAccess.CopyOnWrite };  
+            yield return new object[] { MemoryMappedFileAccess.ReadWriteExecute, MemoryMappedFileAccess.ReadExecute };
+            yield return new object[] { MemoryMappedFileAccess.ReadWriteExecute, MemoryMappedFileAccess.ReadWriteExecute };
+            yield return new object[] { MemoryMappedFileAccess.ReadExecute, MemoryMappedFileAccess.Read };
+            yield return new object[] { MemoryMappedFileAccess.ReadExecute, MemoryMappedFileAccess.CopyOnWrite };
+            if (PlatformDetection.IsNotMacCatalyst)
+            {
+                // https://github.com/dotnet/runtime/issues/114403
```

Comment:
nit: 
Maybe moving the comment above the if, and adding "Skipped on MacCatalyst..." makes it clearer. I don't have strong opinion just find it weird to read. 
Additionally, in the mentioned issue it makes sense to note that the test got disabled in this PR so it is easier to track down when we fix the mentioned issue.